crates/corro-agent/src/agent/tests.rs

```diff
@@ -623,18 +608,15 @@ async fn large_tx_sync() -> eyre::Result<()> {
 
         let res = timeout(
             Duration::from_secs(5),
-            client.request(
-                hyper::Request::builder()
-                    .method(hyper::Method::POST)
-                    .uri(format!("http://{}/v1/transactions", ta1.agent.api_addr()))
-                    .header(hyper::header::CONTENT_TYPE, "application/json")
-                    .body(serde_json::to_vec(&req_body)?.into())?,
-            ),
+            client
+                .post(format!("http://{}/v1/transactions", ta1.agent.api_addr()))
+                .header(hyper::header::CONTENT_TYPE, "application/json")
+                .body(serde_json::to_vec(&req_body)?)
```

Comment:
I didn't do this to stay as close to the old code as possible, but this code could be simplified to use `.json(<Serialize>)` which automatically sets the content_type